### ISSUE TITLE
Some elements lack focus styles

### ISSUE THREAD

**Issue by sjoerddebruin** [wmnl/wikiportrait#92]

**Comment by mbch331** [comment]
Wordt nu opgevolgd in: https://phabricator.wikimedia.org/T214939
